docs/photoniq/vwrs/03-configuration-and-setup/01-configuring-edgeworkers.md

```diff
@@ -0,0 +1,126 @@
+---
+sidebar_position: 30
+title: Configure a VWRs EdgeWorker
+---
+
+# Overview
+
+VWRs require [Akamai EdgeWorker Documentation](https://techdocs.akamai.com/edgeworkers/docs) to execute functions at the edge to help redirect and optimize web traffic for your websites. To get started using EdgeWorkers with VWRs, you need to import the `VirtualWaitingRoom` class from the photoniq-vwrs-client using Node Package Manager (NPM). This integrates the VWRs with the EdgeWorker. 
```

Comment:
@naimishvora-macrometa / @todor-macrometa  do we use Node Package Manager?

Reply:
@joubert-macrometa @maryann-macrometa We don't need NPM. We just copy-paste the file in the EdgeWorker code bundle
![virtualWaitingRoom](https://github.com/Macrometacorp/docs/assets/153493191/a7913805-2467-4b82-b030-427ae8b74f1a)

We are internally using npm to transpile the edgeworker code from TypeScript to this JavaScript file, but the clients shouldn't know or care about this.

Reply:
@joubert-macrometa @maryann-macrometa This is what she has in mind:
https://github.com/Macrometacorp/photoniq-vwrs-client?tab=readme-ov-file#installing-client-library-from-macrometa-private-npm-repository

It explains how to use VWRS's EdgeWorker as a component of another EdgeWorker. 

I think we shouldn't add this docs page for now, especially since `photoniq-vwrs-client` is a private repository (and NPM package).

Reply:
@todor-macrometa pls i need further clarification on this. If I'm getting you correctly, you're saying to remove  everything concerning installing the `photoniq-vwrs-client`?

Reply:
@maryann-macrometa My initial reaction was, if anyone cares about this, they're a programmer, and would consult with the private github repo that they're using, and not the general documentation. But maybe it's better to have it in both places? Your call, either way works for me.

Reply:
@maryann-macrometa Oh, I just noticed this guide for `photoniq-vwrs-client` was already part of the documentation and you've just moved it to another file. Okay, let it stay as it is.